### ISSUE TITLE
Out of memory error on custom text classification data with 49 classes. GPU device - GeForce RTX 2080 Ti. 10986MiB memory.

### ISSUE THREAD

**Issue by spiritbreake** [google-research/uda#34]
I am getting this error when I am trying to execute run_base_uda.sh custom classification data.

ResourceExhaustedError (see above for traceback): OOM when allocating tensor with shape[7168,768] and type float on /job:localhost/replica:0/task:0/device:GPU:0 by allocator GPU_0_bfc
         [[node gradients/bert_1/encoder/layer_7/output/LayerNorm/moments/SquaredDifference_grad/sub (defined at /media/uda/text/bert/optimization.py:168) ]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

It was mentioned in the readme that the code runs fine for 11GB device with max_seq_len=128 and batch_size=8. Any suggestions/instructions to execute the code on GPU would be helpful.

**Comment by qizhex** [comment]
Maybe you can further reduce max_seq_len to 100 since the softmax layer may consume more memory due to more classes.

**Comment by spiritbreake** [comment]
Is there a way to run entire training and testing in tensorflow fp16?
